docs/docs/resources.md

```diff
@@ -0,0 +1,403 @@
+# Resources
+
+_this document has been adapted/copied[^copying] from the Starbeam[^starbeam] documentation_
+
+
+This is a highlevel introduction to Resources, and how to use them.
+For how to integrate Resources in to Ember (Components, etc), see ['./ember.md]('./ember.md'); 
+
+In addition to the live demos accompanying each code snippet, all code snippets will have their starbeam counterparts below them, so that folks can see how similar the libraries are.
```

Comment:
```suggestion
In addition to the live demos accompanying each code snippet, all code snippets will have their Starbeam counterparts below them, so that folks can see how similar the libraries are.
```

Reply:
thanks!